src/js/store/orgs/actions.ts

```diff
@@ -348,3 +353,37 @@ export const commitFailed = ({
     payload: model,
   });
 };
+
+export const refreshOrg = (org: Org): ThunkResult => async (dispatch) => {
+  const url = window.api_urls.scratch_org_refresh(org.id);
+  dispatch({
+    type: 'REFRESH_ORG_STARTED',
```

Comment:
This is where I'd recommend `SCRATCH_ORG_REFRESH_REQUESTED`. We don't actually know if it's started yet.